src/figuresClasses.ts

```diff
@@ -1,19 +1,125 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape;
+  color: Color
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  color: Color;
 
-}
+  shape: Shape;
+
+  sideA: number;
+
+  sideB: number;
+
+  sideC: number;
 
-export class Circle {
+  constructor(
+    color: Color,
+    sideA: number,
+    sideB: number,
+    sideC: number,
+  ) {
+    this.shape = 'triangle';
+    this.color = color;
+    this.sideA = sideA;
+    this.sideB = sideB;
+    this.sideC = sideC;
```

Comment:
Actually there is no need to write this values here 🙂  Just set this fields as public inside constructor arguments

Reply:
Please fix everywhere